android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/StatsIgUtil.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.android.architecture.blueprints.todoapp.util
+
+import android.app.Application
+import com.statsig.androidsdk.DynamicConfig
+import com.statsig.androidsdk.Statsig
+import com.statsig.androidsdk.StatsigUser
+
+
+/**
+ * Object class StatsIgUtil where all Statsig SDK features will be declared and used.
+ */
+object StatsIgUtil {
```

Comment:
I dont think you need this class at all - its very redundant.  You should just call the statsig methods inline, adding this utils class doesnt do much but increase complexity and verbosity

Reply:
Okay I will remove it.